src/Standards/Squiz/Tests/Commenting/FunctionCommentUnitTest.php

```diff
@@ -115,7 +115,7 @@ public function getErrorList()
             997  => 1,
             1004 => 2,
             1006 => 1,
-            1029 => 1,
+            1034 => 1,
```

Comment:
If the tests would stay in the current place, you will also need to adjust the line number referenced below on line 154 (test which only expects an error on PHP < 8.0).